components/org.wso2.carbon.event.outbound.adapter.websubhub/pom.xml

```diff
@@ -0,0 +1,109 @@
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>identity-outbound-adapters</artifactId>
+        <groupId>com.wso2.identity.outbound.adapters</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>org.wso2.carbon.event.outbound.adapter.websubhub</artifactId>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>bundle</packaging>
+    <name>Event Outbound Websubhub Adapter Module</name>
+    <description>org.wso2.carbon.event.outbound.adapter.websubhub provides the back-end functionality of websubhub event adapter
+    </description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.authentication.framework</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
+            <artifactId>httpasyncclient</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.googlecode.json-simple.wso2</groupId>
+            <artifactId>json-simple</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>
+                            org.wso2.carbon.event.outbound.adapter.websubhub.internal,
+                            org.wso2.carbon.event.outbound.adapter.websubhub.internal.*
+                        </Private-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.event.outbound.adapter.websubhub.internal,
+                            !org.wso2.carbon.event.outbound.adapter.websubhub.internal.*,
+                            org.wso2.carbon.event.outbound.adapter.websubhub.*
```

Comment:
we have to specify exact packages that we export.

Reply:
Since we are not exporting only the internal package added in this way. 
All the import packages are done one by one. Is that fine?